archetype/pom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.eclipse.starter</groupId>
+	<artifactId>jakarta-starter</artifactId>
+	<version>2.0-SNAPSHOT</version>
+	<packaging>maven-archetype</packaging>
+
+	<name>Eclipse Starter for Jakarta EE</name>
+	<description>
+		This is the official Eclipse Foundation starter for Jakarta EE.
+		It generates code to help get started with Jakarta EE projects. 
+		It is possible to do so using Maven archetypes. 
+  </description>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Does this project really need to run on java 8? we are at java 19 already and 8 has been deprecated for a while now.

Reply:
I believe we have general agreement on this already. We need to support Jakarta EE 8 and the required baseline for Jakarta EE 8 is Java SE 8. Moreover, in our customer base, Java SE 8 and Jakarta EE 8 remain extremely popular. You can take a look at major reports like this one and see how popular Java SE 8 still is: https://newrelic.com/resources/report/2022-state-of-java-ecosystem.

If we feel this needs more discussion, let's kind start and thread on the mailing list?